src/linux_config_librem5

```diff
@@ -257,7 +260,11 @@ CONFIG_POWER_RESET_VEXPRESS=y
 CONFIG_POWER_RESET_SYSCON=y
 CONFIG_POWER_RESET_SYSCON_POWEROFF=y
 CONFIG_SYSCON_REBOOT_MODE=y
+CONFIG_BATTERY_MAX17042=y
```

Comment:
I wouldn't enable the battery gauge driver until there's a mechanism to choose the correct device tree for used hardware revision in Jumpdrive - otherwise you may end up with the kernel initializing wrong battery capacity data that will persist until the battery is taken out.

Reply:
(other option could be to remove the battery gauge configuration from the device tree used by Jumpdrive)